scripts/macos/RenameMacUserNameAndHomeDirectory.sh

```diff
@@ -23,39 +31,119 @@ fi
 oldUser=$1
 newUser=$2
 
+#Verify valid username
 if [[ -z "${newUser}" ]]
 then
-	abort "New user name must not be empty!"
+	echo "New user name must not be empty!"
```

Comment:
Indentation is off

Reply:
Do we just want to allow this and let it fail later? I'm sure MacOS doesn't allow empty usernames so it should error out anyway?